### PR TITLE
add marks movies

### PR DESCRIPTION
**Pull request by mark7389**
Hey Ab added my favorite movies

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -11,10 +11,17 @@
...
     <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
     <!-- end CSS stylesheetss -->
-    
+
     <title>Favorite Movies</title>
   </head>
   <body>
     <h1> Favorite Movies </h1>
+
+    <h2>Mark's Favorite Movies</h2>
```
Comment (serpentz) [inline index.html]: This line is redundant, I think we should remove it.

File: index.html [hunk 2]
```diff
@@ -11,10 +11,17 @@
...
+    <ul>
```
Comment (serpentz) [inline index.html]: can we add bootstrap classes here?

File: index.html (hunk 1 above)
Comment (mark7389) [reply index.html]: Oops - will do :+1:

File: index.html (hunk 2 above)
Comment (mark7389) [reply index.html]: :dancer:

**Pull request merged**